lintly/parsers.py

```diff
@@ -330,6 +329,56 @@ def parse_violations(self, output):
 
         return violations
 
+class GitLeaksParser(BaseLintParser):
+    """
+    Gitleaks JSON format
+
+        {
+            "line": "-----BEGIN PRIVATE KEY-----",
+            "lineNumber": 59,
+            "offender": "-----BEGIN PRIVATE KEY-----",
+            "commit": "111111111111111111111000000000",
+            "repo": ".",
+            "repoURL": "",
+            "leakURL": "",
+            "rule": "Asymmetric Private Key",
+            "commitMessage": "any commit message \n",
+            "author": "bob s",
+            "email": "bob@example.com",
+            "file": "relative/path/to/output",
+            "date": "2020-04-14T15:17:53-07:00",
+            "tags": "key, AsymmetricPrivateKey"
+            }
```

Comment:
Being super nitpicky here but the closing bracket should be at the same indentation level as the opening bracket on line 336.

Reply:
fixed in a force-push.